services/vision/mlvision/ml_model.go

```diff
@@ -69,18 +69,31 @@ func registerMLModelVisionService(
 	if err != nil {
 		return nil, err
 	}
+
 	classifierFunc, err := attemptToBuildClassifier(mlm)
 	if err != nil {
-		logger.Infow("error turning turn ml model into a classifier:", "model", params.ModelName, "error", err)
+		logger.Infof("%v", errors.Wrapf(err, "was not able to turn ml model %q into a classifier", params.ModelName))
```

Comment:
Keep this as Infow in order to allow the logger to format the information as it sees fit